lib/src/main/java/com/auth0/android/lock/utils/LoadCountriesTask.java

```diff
@@ -26,53 +26,48 @@
 
 import android.content.Context;
 import android.os.AsyncTask;
-import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
 import android.util.Log;
 
+import androidx.annotation.NonNull;
+
+import com.auth0.android.lock.adapters.Country;
 import com.google.gson.Gson;
 import com.google.gson.reflect.TypeToken;
 
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.io.Reader;
 import java.lang.reflect.Type;
+import java.util.ArrayList;
 import java.util.Collections;
+import java.util.List;
 import java.util.Map;
 
-public abstract class LoadCountriesTask extends AsyncTask<String, Void, Map<String, String>> {
+public abstract class LoadCountriesTask extends AsyncTask<Context, Void, List<Country>> {
 
     private static final String TAG = LoadCountriesTask.class.getName();
-    public static final String COUNTRIES_JSON_FILE = "com_auth0_lock_passwordless_countries.json";
-
-    private final ThreadLocal<Context> context = new ThreadLocal<>();
```

Comment:
this reference changed at some point, making the task not to execute at all. I'm passing this as a "task param" rather than via the constructor